docs/notes/bugfix-20792.md

```diff
@@ -0,0 +1 @@
+[[ Bug 20792 ]] Send touch messages when android scroller `scrollingEnabled` is `false`
```

Comment:
You've used the commit message format for the bugfix note

Reply:
woops!

Reply:
fixed